sched/group/group_signal.c

```diff
@@ -109,7 +111,8 @@ static int group_signal_handler(pid_t pid, FAR void *arg)
        * probably blocked).
        */
 
-      if (sigismember(&tcb->sigwaitmask, info->siginfo->si_signo) && !info->atcb)
+      if (nxsig_ismember(&tcb->sigwaitmask, info->siginfo->si_signo) &&
```

Comment:
```suggestion
      ret = nxsig_ismember(&tcb->sigwaitmask, info->siginfo->si_signo);
      if (ret == 1 &&
```

Reply:
Hmmm... In this case the signal number should be okay. Probably a DEBUGASSERT(GOOD_SIGNO( info->siginfo->si_signo)) would give us sufficient protection.

Reply:
I used your suggested code rather than DEBUGASSERT.